articles/redis-cache/cache-faq.md

```diff
@@ -388,7 +388,10 @@ How to configure this setting:
   > The value specified in this configuration element is a *per-core* setting. For example, if you have a 4 core machine and want your minIOThreads setting to be 200 at runtime, you would use `<processModel minIoThreads="50"/>`.
   >
 
-* Outside of ASP.NET, use the [ThreadPool.SetMinThreads(…)](https://msdn.microsoft.com/library/system.threading.threadpool.setminthreads.aspx) API.
+* Using [ThreadPool.SetMinThreads (...)] (https://msdn.microsoft.com/library/system.threading.threadpool.setminthreads.aspx) API outside ASP.NET and Azure WebSites global.asax To do.
```

Comment:
Wording here should be adjusted some:

* Outside ASP.NET, and Azure WebSites global.asax, use the [ThreadPool.SetMinThreads (...)] (https://msdn.microsoft.com/library/system.threading.threadpool.setminthreads.aspx) API.